.env.example

```diff
@@ -26,6 +26,7 @@ BUILDER_SERVER_URL=
 BUILDER_SHARE_URL=
 
 COLLECTIONS_GRAPH_URL=
+GRAPH_QUERY_TIMEOUT=
```

Comment:
Did you change the builder's definition file to add the environment variable as well?

Reply:
My bad, already saw it.